roadmap/2022.md

```diff
@@ -54,3 +54,29 @@ Misc:
   - [ ] Finish changes regarding the API keys
   - [ ] Finish changes regarding the new index routes
   - [ ] Add the tenant tokens methods
+
+## Q2
+
+Analytics:
+- [ ] Finish the implementation in the remaining SDKs (Java, Javascript, Rust)
+- [ ] Plan how we will implement the analytics on the integrations, tools & plugins.
+- [ ] _(may not be required)_ Define the specs to support analytics in some tools with core team.
+- [ ] Implement it on the integrations.
+- [ ] Implement it on the plugins.
```

Comment:
```suggestion
- [ ] Implement it on the all the integrations maintained by the integration team.
```

Reply:
I'm not sure what you mean by "plugin" regarding [this](https://github.com/meilisearch/integration-guides#-our-integrations)
You need to implement it in
- web framework integration
- front-end integration
- platform plugin
- devOps tools (kubernetes only)
- the other integrations without any category: docs-scaper

Reply:
Oh yeah, I meant "platform plugins". But I like the non-sdk suggestion from @bidoubiwa.

But to be honest, I put that in different items to have a sense of progress and at the same time, I don't want to specify all of the items individually. 
So I'll accept your suggestion :)

Reply:
If we have not finished all the implementation by the end of the quarter, we could add the list of the already finished ones :)